Makefile

```diff
@@ -42,7 +42,7 @@ reason: setup
 
 .PHONY: clean cleanbisect
 clean: cleanbisect
-	rm -rf _build dune-workspace
+	rm -rf _build dune-workspace src/*[0-9]*.ml
```

Comment:
I am uncomfortable with such wildcards in arguments to rm. It would be easy to add a source file that this would delete.